megamek/resources/log4j2-release.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration name="release-configuration" status="off">
+
+    <!-- This is the release log4j2 configuration, intended for when end users run the app. -->
+
+    <properties>
+        <property name="logfile.filename">${sys:megamek.log.file:-./logs/mm.log}</property>
+        <property name="logfile.pattern">%8relative [%-30.30thread] %-5level %-40.40logger - %message{nolookups}%n%rThrowable</property>
+    </properties>
+
+    <appenders>
+        <file name="file" fileName="${logfile.filename}" append="false" createondemand="true">
+            <patternlayout pattern="${logfile.pattern}" disableAnsi="true" />
+        </file>
+    </appenders>
+
+    <loggers>
+        <logger name="megamek" level="info" />
+        <logger name="System"  level="info" />
+        <root level="warning">
+            <appenderref ref="console" />
```

Comment:
2018-09-08 10:31:06,955 main ERROR Unable to locate appender "console" for logger config "root"

Reply:
Great catch! (both of these) - I forgot to actually try and run MM from the jar.

(in case you want to fix it locally to go on with the testing, `<root level="warning">` should be `<root level="warn">` and the `<appenderref ref="console" />` should be removed – I'll test and update the PR anyways).